test/test_lang.cc

```diff
@@ -155,6 +157,8 @@ void testTcFormat() {
   auto def_tree = Parser(source).parseFunction();
   std::ostringstream s;
   tcFormat(s, def_tree);
+  LOG(INFO) << s.str();
```

Comment:
Tests shouldn't print, just test.
Is this debug information you forgot to clean up?
Looks like this file should be untouched.